fastlane/Fastfile

```diff
@@ -19,40 +19,62 @@ def set_bundle_version_from_commit_count
   )
 end
 
-def clear_bundle_version
-  puts "removing CFBundleVersion..."
+# This will keep a copy of the original Info.plist
+@original_info_plist = nil
+
+def save_original_info_plist
   update_info_plist(
     xcodeproj: "ios/Exponent.xcodeproj",
     plist_path: "Exponent/Supporting/Info.plist",
     block: proc do |plist|
-      plist.delete("CFBundleVersion")
+      @original_info_plist = Marshal.load(Marshal.dump(plist))
     end
   )
 end
 
-# TODO: is there a way to modify the plist and automatically revert the changes
-# after the build?
-def configure_for_app_store
-  puts "configuring app for App Store submission..."
-  puts "removing background location permission strings and background mode..."
+def restore_original_info_plist
   update_info_plist(
     xcodeproj: "ios/Exponent.xcodeproj",
     plist_path: "Exponent/Supporting/Info.plist",
     block: proc do |plist|
-      plist.delete("NSLocationAlwaysUsageDescription")
+      puts "Restoring original Info.plist..."
+      plist.replace(@original_info_plist)
+    end
+  )
+end
+
+def remove_background_location
+  update_info_plist(
+    xcodeproj: "ios/Exponent.xcodeproj",
+    plist_path: "Exponent/Supporting/Info.plist",
+    block: proc do |plist|
```

Comment:
Instead of the block, this might be simpler: https://docs.fastlane.tools/actions/set_info_plist_value/